eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -26,12 +28,15 @@ stages:
                 deploy:
                   steps:
                     - checkout: self
+                    - template: /eng/pipelines/templates/steps/stage-artifacts.yml
+                      parameters:
+                        SourceFolder: ${{parameters.ArtifactName}}
+                        TargetFolder: ${{artifact.safeName}}
+                        PackageName: ${{artifact.name}}
                     - pwsh: |
-                        Get-ChildItem $(Pipeline.Workspace)/${{parameters.ArtifactName}}
-                        New-Item -Type Directory -Name ${{artifact.safeName}} -Path $(Pipeline.Workspace)
-                        Copy-Item $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}-[0-9]*.[0-9]*.[0-9]* $(Pipeline.Workspace)/${{artifact.safeName}}
-                        Get-ChildItem $(Pipeline.Workspace)/${{artifact.safeName}}
-                      displayName: Stage artifacts
+                        Get-ChildItem -Recurse $(Pipeline.Workspace)/${{artifact.safeName}}
```

Comment:
Why do we need this get-childitem?

Reply:
Because I was copy pasting the script. 👍 

I'll clean it up on my next pass through. I manually added a `output visible artifacts` anyway.